File: src/rails/game/state/UnknownOwner.java
package rails.game.state;

/**
 * Used to initialize ownable items with an owner
 * PortfolioManager creates a Singleton of this class
 */
public class UnknownOwner extends AbstractItem implements Owner, DelayedItem {

    private UnknownOwner(Item parent, String id) {
        super(parent, id);
    }

    static UnknownOwner create(Item parent, String id) {
        return new UnknownOwner(parent, id);
    }
    
}

File: dotnet/RailsLib.Net/Net/Game/State/UnknownOwner.cs
using System;

/**
 * Used to initialize ownable items with an owner
 * PortfolioManager creates a Singleton of this class
 */

namespace GameLib.Net.Game.State
{
    public class UnknownOwner : AbstractItem, IOwner
    {
        private UnknownOwner(IItem parent, string id) : base(parent, id)
        {
        }

        public static UnknownOwner Create(IItem parent, string id)
        {
            return new UnknownOwner(parent, id);
        }
    }
}
